www/janf04/semestralka1/fotogalerie.html

```diff
@@ -9,20 +9,20 @@
     <meta name="description" content="Fotbalový klub Galacticos FC vznikl v roce 2016. Aktuálně hrajeme ligu PKFL.">
     <meta name="keywords" content="fotbal, gol, míč, stránky, klub, hráči, zápas">
     <link rel="shortcut icon" type="image/ico" href="pics/favicon.ico"/>
-    <link rel="stylesheet" type="text/css" media="screen" href="styles/main.css" />
+    <link rel="stylesheet" type="text/css" href="styles/main.css" />
     <script src="main.js"></script>
 </head>
 
 <body>
     <header>
         <div class="logo">
-            <h1>Galacticos Praha</h1>
+            <h1>Galacticos FC</h1>
         </div>
         <nav>
             <a href="index.html">Aktuality</a>
             <a href="zapasy.html">Zápasy</a>
             <a href="hraci.html">Hráči</a>
-            <a href="fotogalerie.html">Fotogalerie</a>
+            <a class="current" href="fotogalerie.html"><span background-color="grey">Fotogalerie</span></a>
```

Comment:
nevalidni, neni to ani css ani nic

Reply:
jo ok, akorat to muze byt jeste vyrazneji odliseno

Reply:
hotovo